src/js/content.js

```diff
@@ -22,15 +22,15 @@ chrome.runtime.onConnect.addListener(function(port) {
   }
   port.onDisconnect.addListener(() => {
     messenger.stop()
-    document.removeEventListener("click", clickHandler, true)
+    window.removeEventListener("click", clickHandler, true)
```

Comment:
This solves the issue of the closing card after clicking the MOCO bubble, probably because it is called earlier when attached to the top most object.